sup3r/bias/bias_calc.py

```diff
@@ -91,16 +101,22 @@ def __init__(self, base_fps, bias_fps, base_dset, bias_feature,
             self.bias_fps = sorted(glob(self.bias_fps))
 
         self.base_handler = getattr(rex, base_handler)
-        self.bias_handler = getattr(sup3r.preprocessing.data_handling,
-                                    bias_handler)
+        self.bias_handler = getattr(
+            sup3r.preprocessing.data_handling, bias_handler
+        )
 
         with self.base_handler(self.base_fps[0]) as res:
             self.base_meta = res.meta
             self.base_tree = KDTree(self.base_meta[['latitude', 'longitude']])
 
-        self.bias_dh = self.bias_handler(self.bias_fps, [self.bias_feature],
-                                         target=self.target, shape=self.shape,
-                                         val_split=0.0, **bias_handler_kwargs)
+        self.bias_dh = self.bias_handler(
+            self.bias_fps,
+            [self.bias_feature],
+            target=self.target,
+            shape=self.shape,
+            val_split=0.0,
+            **bias_handler_kwargs,
+        )
```

Comment:
Did you run an auto formatter on everything? Do you like this style better and why? Some thoughts: The original formatting has strong horizontal separation of the output var, method, and input args. In the original formatting, it's very clear which lines are part of the function call. In the new formatting, it's more difficult to visually separate the output arg from the input args. Lastly, what is the logic for having the closing parenthesis on the 0 column? It's closing the function call, not the original line?

Reply:
@bnb32 it looks like some of the files are still black-formatted. Are you going to revert this or are we going with the black style?

Reply:
I just think some formatting like in the screenshot below is total garbage (forward_pass.py). It looks like C++ nonsense and i think we're losing the readability of python. Some of this stuff is just the result of chaining too many commands into one "line". 

<img width="598" alt="image" src="https://github.com/NREL/sup3r/assets/12700356/524b1ac9-34f3-401e-892a-b4031685637a">

Reply:
Yeah I'm not a fan of that formatting either but I don't think it's that frequent and I personally consider the consistency a net positive. I think 
```out_arg = func(
   input_args
)``` is quite readable in general though.

Reply:
I think it's a step backwards as far as code standards but if you want to do it for this repo i'll leave that decision to you. I think it's worst with massive chained "one-liners" like in the screenshot above. Basically i think this moves the onus to the developer to write smaller function chains to get more sensible code (e.g., split up the code above into 3-4 single lines that are actually single lines). IMO we should have better standards than in the example above. If we do want to do black across the board, i recommend adding the github action: https://github.com/marketplace/actions/run-black-formatter

Reply:
Do you think that onus is a bad one? I think the example above is the worst and for 99% the readability is maintained or improved tbh. I've turned off black formatting though since I don't want to enforce something you don't like, but I'd prefer not to have to manually revert a ton.

Reply:
I don't think moving the focus onto a different type of style is bad. I do think black makes some code more readable and some other code much less so. I guess it just goes to show that these tools won't automatically transform any code into "good" code. There's still a lot of human considerations the developer has to work with to make readable code.

Reply:
Definitely agree!